DigitalLearningSolutions.Web/Styles/trackingSystem/centreAdministrators.scss

```diff
@@ -0,0 +1,31 @@
+﻿@import "~nhsuk-frontend/packages/core/all";
+
+button.nhsuk-pagination__link {
+  @include mq($until: 1024px) {
+    flex-direction: column;

```

Comment:
I'm assuming the flex direction here is actually "--next" style specific (being reverse of prev)?

Reply:
I suppose technically yes. Will make it explicit